src/app/productManagement/relatedProducts/templates/relatedProducts.html

```diff
@@ -0,0 +1,69 @@
+<div class="row">
+    <div class="col-sm-12">
+        <h4>Related Products  <span class="badge">{{relatedProduct.relatedProducts.Meta.TotalCount || 0}}</span></h4>
+        <div class="row">
+            <div class="col-sm-8 col-md-9">
+                <ui-select multiple ng-model="relatedProduct.selectedProducts" close-on-select="false">
+                    <ui-select-match placeholder="Search and select Products to add...">
+                        {{$item.Name}}
+                        <span ng-bind="$select.selected.Name"></span>
+                    </ui-select-match>
+                    <ui-select-choices repeat="product in (relatedProduct.uiSelectProducts.Items | filter: $select.search) track by $index" refresh="relatedProduct.listAllProducts($select.search)" refresh-delay="300">
```

Comment:
I would actually prefer this to be a full on product list table with an assignment checkbox for marking them as related. Filter out the current product from the list too.